nemoguardrails/library/autoguard/README.md

```diff
@@ -4,92 +4,216 @@ This package implements the AutoGuard API integration.
 
 AutoGuard comes with a library of built-in guardrails that you can easily use:
 
-1. [Confidential Detection](#confidential-detection)
-2. [Gender bias Detection](#gender-bias-detection)
-3. [Harm Detection](#harm-detection)
-4. [Toxicity detection](#toxicity-detection)
+1. [Gender bias Detection](#gender-bias-detection)
+2. [Harm Detection](#harm-detection)
+3. [Jailbreak Detection](#jailbreak-detection)
+4. [Confidential Detection](#confidential-detection)
 5. [Racial bias Detection](#racial-bias-detection)
-6. [Jailbreak Detection](#jailbreak-detection)
-7. Factcheck
-8. PII
+6. [Tonal Detection](#tonal-detection)
+7. [Toxicity detection](#toxicity-extraction)
+8. [PII](#pii)
+9. [Factcheck](#factcheck)
+
+
+Note: Factcheck and PII are implemented a bit differently, compared to other guardrails.
```

Comment:
can you go through the api doc here https://docs.google.com/document/d/1B4_5JZ0BcX9y95DMULMftripYTWh5z-Nh89pfWgYJAE/edit?usp=sharing and see if you can add any guardrail configuration info/language (especially for pii/fact checking) that may not be present in this readme?